charts/kserve-resources/templates/localmodelnode/deployment.yaml

```diff
@@ -45,12 +45,25 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+        volumeMounts:
+          - mountPath: /models
+            name: local-models
+            readOnly: false
         resources:
           limits:
             cpu: 100m
             memory: 300Mi
           requests:
             cpu: 100m
             memory: 200Mi
+      volumes:
+        - name: local-models
+          hostPath:
+            path: /mnt/models
+            type: Directory
```

Comment:
https://kubernetes.io/docs/concepts/storage/volumes/#hostpath-volume-types
type: DirectoryOrCreate because we don't have this folder

Reply:
We also need to test if `fsgroupChangepolicy` works for hostpath. i.e. it changes the ownership of the path to a non-root user.